### ISSUE TITLE
fails to build due to process_read_dir argument error

### ISSUE THREAD

**Issue by stevensonmt** [Byron/dua-cli#72]
```
error[E0593]: closure is expected to take 4 arguments, but it takes 2 arguments
   --> /home/matt/.cargo/registry/src/github.com-1ecc6299db9ec823/dua-cli-2.10.7/src/common.rs:139:14
    |
139 |             .process_read_dir(|_, dir_entry_results| {
    |              ^^^^^^^^^^^^^^^^ ---------------------- takes 2 arguments
    |              |
    |              expected closure that takes 4 arguments
```

This appears to be due to a breaking change in jwalk between version 0.5.0 and 0.5.2.

**Comment by Byron** [comment]
Thanks a lot for the headsup - I have created an issue in [jwalk](https://github.com/jessegrosjean/jwalk/issues/28) which I believe is a better place for the fix - this probably is an accident.

**Comment by stevensonmt** [comment]
It is weird though b/c your Cargo.toml explicitly calls for 0.5.0, so I couldn't figure out why I was getting that error when trying to build.

**Comment by Byron** [comment]
I think by default the lock file is ignored unless some flag is provided to enforce it.
That certainly is a little surprising and I would expect it to be the default.

**Comment by DianaNites** [comment]
Jwalk yanked the offending version, and the build fails again?

```rust
error[E0593]: closure is expected to take 2 arguments, but it takes 4 arguments
   --> .cargo/registry/src/github.com-1ecc6299db9ec823/dua-cli-2.10.8/src/common.rs:139:14
    |
139 |             .process_read_dir(|_, _, _, dir_entry_results| {
    |              ^^^^^^^^^^^^^^^^ ---------------------------- takes 4 arguments
    |              |
    |              expected closure that takes 2 arguments
```

**Comment by Byron** [comment]
Thanks for the heads-up - fixed in v2.10.9.
